configuration/redis/redis.go

```diff
@@ -324,19 +327,30 @@ func (r *ConfigurationStore) handleSubscribedChange(ctx context.Context, req *co
 		return
 	}
 
-	// get all keys if only one is changed
-	getResponse, err := r.Get(ctx, &configuration.GetRequest{
-		Metadata: req.Metadata,
-		Keys:     []string{targetKey},
-	})
-	if err != nil {
-		r.logger.Errorf("get response from redis failed: %s", err)
-		return
+	operation := msg.Payload
+	var items map[string]*configuration.Item
+
+	if operation == "del" {
```

Comment:
why we are checking for del operation? AFAIK config api does not support `del` ops.

Reply:
Config API doesn't support delete as well as add/update keys. But we expect notification after subscribing if a key is updated/deleted. Currently if a redis subscribed key is deleted, it errors out in redis as it tries to `get` the keyValue but it is deleted. So it needs separate handling